src/main/java/Duke.java

```diff
@@ -1,10 +1,244 @@
+import java.util.ArrayList;
+import java.util.Locale;
+import java.util.Scanner;
+<<<<<<< HEAD
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+=======
+import java.io.File;
+import java.io.IOException;
+>>>>>>> branch-Level-7
+
+import static java.lang.Integer.parseInt;
+
 public class Duke {
+    //@@author IceFire
+    //Reused from https://stackoverflow.com/questions/36514289
+    // with minor modifications
+    private static String dashedLine()
+    {
+        StringBuilder sb = new StringBuilder(90);
+        for (int i = 0; i < 90; i++) {
+            sb.append("-");
+        }
+        return sb.toString();
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
+        /*String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+                + "|____/ \\__,_|_|\\_\\___|\n";*/
+
+        Scanner userInput = new Scanner(System.in);
+        System.out.println(dashedLine());
+        System.out.println("My name is Skyler White yo \n"
+                + "How can I help you?");
+        System.out.println(dashedLine());
+        ArrayList<Task> list = new ArrayList<>();
+        boolean end = false;
+        while(!end) {
+            System.out.print("Please Input: ");
+            end = processInput(userInput.nextLine(), list, end);
+        }
+    }
+
+    private static boolean processInput(String input, ArrayList<Task> list, boolean end) {
```

Comment:
It does not seem like there's any need for `end` to be passed in here. Perhaps just initializing and returning it at `case "bye":` is enough?